### ISSUE TITLE
Visual design: Change section header font color to black

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#2848]
Change section header font color to black. It is currently white and color contrast against the beige section header is not accessible.
![image](https://user-images.githubusercontent.com/42216982/124664816-d2cfea00-de60-11eb-9489-98fa147f1c2f.png)

**Comment by justinlittman** [comment]
Dupe of #3076
